doc/releases/changelog-0.25.0.md

```diff
@@ -20,84 +20,69 @@
   [(#2944)](https://github.com/PennyLaneAI/pennylane/pull/2944)
   [(#2644)](https://github.com/PennyLaneAI/pennylane/pull/2644)
 
-  The new [`qml.resource`](https://pennylane.readthedocs.io/en/stable/code/qml_resource.html) module allows you to estimate the number of 
+  The new [resource](https://pennylane.readthedocs.io/en/stable/code/qml_resource.html) module allows you to estimate the number of 
   non-[Clifford gates](https://en.wikipedia.org/wiki/Clifford_gates) and logical 
   qubits needed to implement [quantum phase estimation](https://codebook.xanadu.ai/P.1) 
   algorithms for simulating materials and molecules. This includes support for quantum 
   algorithms using [first](https://en.wikipedia.org/wiki/First_quantization) and [second](https://en.wikipedia.org/wiki/Second_quantization) quantization with specific bases:
 
-  - [First quantization](https://en.wikipedia.org/wiki/First_quantization) using a plane-wave basis via the [`FirstQuantization`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.resource.FirstQuantization.html) class:
+  - [First quantization](https://en.wikipedia.org/wiki/First_quantization) using a plane-wave basis via the `FirstQuantization` class:
 
-  ```python
-  import pennylane as qml
-  from pennylane import numpy as np
-  
-  n = 100000        # number of plane waves
-  eta = 156         # number of electrons
-  omega = 1145.166  # unit cell volume in atomic units
-  
-  algo = FirstQuantization(n, eta, omega)
-  ```
-  
-  ```pycon
-  >>> print(algo.gates, algo.qubits)
-  1.10e+13, 4416
-  ```
+    ```pycon
+    >>> n = 100000        # number of plane waves
+    >>> eta = 156         # number of electrons
+    >>> omega = 1145.166  # unit cell volume in atomic units
+    >>> algo = FirstQuantization(n, eta, omega)
+    >>> algo.gates
+    1.10e+13
+    >>> algo.qubits
+    4416
+    ```
```

Comment:
The formatting here lacks coherence with the `DoubleFactorization` example, that's why I changed it.

Reply:
See new update. Is that better?

Reply:
I think the pycon should be separated in python and pycon like the second example, but it is really minor.